concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1122,8 +1122,8 @@ jobs:
             image_resource:
               type: docker-image
               source:
-                repository: governmentpaas/spruce
-                tag: 465642da06051a55630d39c899697b678f66a7f7
+                repository: keymon/bosh-cli-v2
```

Comment:
Could you rebase these container changes (and one below) out? I found them confusing when reviewing the individual commits and they might confuse somebody looking at the history.